plugin/xtrabackup/plugin.go

```diff
@@ -76,8 +79,9 @@ package main
 
 import (
 	"os"
-
 	fmt "github.com/jhunt/go-ansi"
+	"path/filepath"
+        "syscall"
```

Comment:
You should `go fmt` this file.

Also, we group our imports into three sections:

```
import (
  /* standard library imports */

  /* external libraries */

  /* modules internal to SHIELD (but still on Github) */
)
```

Each group should be separated by blank lines (two `\n`'s)